.changeset/chilled-snakes-thank.md

```diff
@@ -0,0 +1,17 @@
+---
+"@commercetools-frontend/application-shell": minor	
+---
+
+feat(app-shell): add ability to `skipTokenRetry` via context.
+
+The commercetools Platform token is renewed once whenever it expired. This is an automatic mechanism which retries a failed request responding with `401` once based on the GraphQL target.
+
+It is however useful for some requests to disable this mechanism to avoid uncalled-for network requests. We introduced a `skipTokenRetry` property on the GraphQL `context` for this.
```

Comment:
```suggestion
However, it is useful for some requests to disable this mechanism to avoid uncalled-for network requests. This is done by specifying a `skipTokenRetry` property on the Apollo query `context` object.
```